server/views/pages/reviewsTable.njk

```diff
@@ -73,6 +73,14 @@
       {% endif %}
     {% endset %}
 
+    {% set daysSinceLastReview %}
+      {% if review.daysSinceLastReview is null %}
+        Not reviewed
```

Comment:
Shown whenever there has not been a "real" in-person review. This is normally shown only for people who are new to prison, but it's possible in other situations (e.g. IEP records were deleted in NOMIS).